services/movie_session.py

```diff
@@ -25,20 +31,26 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
 
 
 def update_movie_session(
-    session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+        session_id: int,
+        show_time: datetime = None,
```

Comment:
Optional[datetime]

Reply:
> Optional[datetime]

![optional er](https://user-images.githubusercontent.com/120128574/218736705-67b334d5-0dd5-4977-b1f5-7d4d097b64e7.png)
I face this error( how to deal with it?